src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -29,19 +29,16 @@
 import java.awt.event.ActionListener;
 import java.awt.event.KeyEvent;
 import java.awt.event.KeyListener;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.UncheckedIOException;
 import java.time.Duration;
 import java.util.EventListener;
 import java.util.List;
+import java.util.Scanner;
 
-import javax.swing.JButton;
-import javax.swing.JComboBox;
-import javax.swing.JDialog;
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.JPasswordField;
-import javax.swing.JTextField;
-import javax.swing.SpringLayout;
+import javax.swing.*;
```

Comment:
The project standards do not allow package level imports.

Reply:
Fixed..... my IDE is preset to import the package!! how I can change it?

Reply:
This is how I do it in IntelliJ

![image](https://github.com/informatici/openhospital-gui/assets/1105445/6ebaa6db-17c2-4680-89b8-6ff07e98eb14)

Reply:
is this solvable with the `.ide-settings/idea/OpenHospital-code-style-configuration.xml`?